app/src/Api.tsx

```diff
@@ -127,20 +127,14 @@ export class FirebaseService {
     }
   }
 
-  /**
-   * Get user's number of completed tasks by borough
-   * 
-   * @param userId 
-   * @param borough 
-   */
-  async getCompletedTasksByBorough(userId: string, borough: string): Promise<number | null> {
+  async getTasksByBorough(userId: string, borough: string): Promise<any[]> {
```

Comment:
I changed this method to just return the user's tasks by borough instead of just the number bc we need the tasks for the task list page